third_party/nvfuser/csrc/scheduler/utils.cpp

```diff
@@ -1149,13 +1149,7 @@ IterDomain* innerMostRootDim(TensorView* tv) {
     // vectorization opportunity. If we're looking at a reduction reference
     // tensor we want to consider the reduction iteration domains as domains we
     // can vectorize on.
-    if ((*it)->isReduction() && tv->isFusionInput()) {
-      continue;
-    }
-    if ((*it)->isBroadcast()) {
-      if (inner_most_id == nullptr) {
-        inner_most_id = *it;
-      }
```

Comment:
I am not sure either. But this would only happen on all-broadcasting tensors, and this function is only used in vectorization and transpose detection. Loosely speaking, if a tensor is all-broadcasting, it should be a no-op in vectorization/transpose analysis, so I would speculate this change is safe.